VariantSurvival_package/R/VariantSurvival.R

```diff
@@ -1,369 +1,740 @@
+                          margin-top: 210px; 
+                          margin-bottom: 20px;", 
+                          shiny::h4(
+                            "Life table", 
+                            style = "margin: 0; flex-grow: 1;"
+                            )
+                          ),
+                          shinydashboard::tabBox(
+                            id = "myBox_mm",
+                            title = "",
+                            width = 12,
+                            shiny::tabPanel(
+                              "with",
+                              shiny::span(
+                                DT::dataTableOutput("lt_mm_0")
                               )
                             ),
-                            fluidRow(column(
-                              12,
-                              div(style = "height:200px; Bottomleft"),
-                              checkboxInput("life_table_null_model",
-                                            "Display life table",
-                                            value = FALSE),
-                              box(
-                                id = "myBox",
-                                title = "",
-                                width = '200px',
-                                span(DT::dataTableOutput("null_model_life_table"))
+                            shiny::tabPanel(
+                              "without",
+                              shiny::span(
+                                DT::dataTableOutput("lt_mm_1")
                               )
-                            )))
-                 )
-               ))),
-      tabPanel(
+                            )
+                          )
+                        )
+                      )
+                    )
+                  ),
+                  shiny::column(
+                    width = 4,
+                    shiny::div(
+                      style = "display: flex; 
+                      align-items:
+                      center;
+                      margin-bottom: 10px;",  
+                      shiny::h4(
+                        "P-Values Table", 
+                        style = "margin: 0; flex-grow: 1;"
+                        ),  
+                      shiny::actionButton(
+                        "infoBtn",
+                        label = NULL, 
+                        icon = shiny::icon("info-circle"), 
+                        style = "margin-left: 3px;
+                                background: transparent; 
+                                 border: none; 
+                                 color: blue; 
+                                 cursor: pointer;"
+                        ) 
+                    ),
+                    # P-values table
+                    DT::dataTableOutput("p_values_km")
+                  )
+                )
+              ),
+              shiny::tabPanel(
+                title = span("Null Model", 
```

Comment:
_span()_ requires scope (found multiple ones for function _span_, don't know which one is required here)

Reply:
sometimes _htmltools::_, sometimes _shiny::_